src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -267,75 +320,108 @@ public static Bundle convertToFHIR(Person person, long stopTime) {
     for (Encounter encounter : person.record.encounters) {
       BundleEntryComponent encounterEntry = encounter(person, personEntry, bundle, encounter);
 
-      for (HealthRecord.Entry condition : encounter.conditions) {
-        condition(person, personEntry, bundle, encounterEntry, condition);
+      if (shouldExport("Condition")) {
```

Comment:
If you take my suggestion above about using classes instead of strings, this would become `shouldExport(Condition.class)` which would make it less prone to typo errors.